README.md

```diff
@@ -4,26 +4,30 @@
 
 A **work-in-progress** [Magit](https://magit.vc) clone for [Neovim](https://neovim.io) that is geared toward the Vim philosophy.
 
+## Notice
+
+Neogit has moved to an organization at <https://github.com/NeogitOrg/neogit/issues> to ensure the longevity of this project and ensure that it is more accessible to collaborators.
+
 ## Installation
 
-**NOTE**: We depend on [plenary.nvim](https://github.com/nvim-lua/plenary.nvim), so to use this plugin, you will additionally need to
-require `nvim-lua/plenary.nvim` using your plugin manager of choice, before requiring this plugin.
+**NOTE**: We depend on [plenary.nvim](https://github.com/nvim-lua/plenary.nvim) and, optionally, [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim), so to use this plugin, you will additionally need to

```

Comment:
This is good. Perhaps mention "if enabled by `use_telescope=true`" or similar.

I've seen plugins use an approach where they auto-detect the presence of other plugins, and I think some people will expect that and find that it doesn't work.

Reply:
Good idea

https://github.com/NeogitOrg/neogit/issues/550